Gulpfile.js

```diff
@@ -9,23 +9,59 @@
 //   no-op copy task. This allows other packages that depend on yours to contain
 //   a copy task without failing the gulp build.
 
+/*
```

Comment:
dude i've tried to `ts`-ify the gulp build several times! it's mainly time-consuming. i think VSCode's [JS IntelliSense](https://code.visualstudio.com/Docs/languages/javascript#_intellisense) would actually get us most of the way there...